src/System.Private.ServiceModel/tests/Common/Infrastructure/src/CertificateManager.cs

```diff
@@ -73,7 +73,7 @@ private static StoreLocation PlatformSpecificRootStoreLocation
                 using (X509Store store = new X509Store(storeName, storeLocation))
                 {
                     store.Open(OpenFlags.ReadOnly);
-                    resultCert = CertificateFromThumbprint(store, certificate.Thumbprint);
+                    resultCert = CertificateFromThumbprint(store, certificate.Thumbprint, validOnly: false);
```

Comment:
Reviewers: I altered this to allow invalid certs to be retrieved.  We are seeing issues on Linux where the certs are considered invalid, and it caused the auto-detection to disable all cert tests.  It also meant the CI and lab machines would not be able to force the certs to be loaded by manually setting the TestProperties to enable the cert tests. I felt it was better to always allow the cert to be loaded if possible, and let tests pass or fail based on cert validity.

The real test that the certs are valid now happens in the Infrastructure scenario tests for certs.